### PR TITLE
Add direction field to harfanglab parser

### PR DESCRIPTION
**Pull request by ADE-dstny**
Adding the direction field on network event

### CODE REVIEW

File: HarfangLab/harfanglab/ingest/parser.yml [hunk 1]
```diff
@@ -82,6 +82,9 @@ stages:
       - set:
           destination.port: "{{json_event.message.dport}}"
 
+      - set:
+          network.direction: "{{json_event.message.direction}}"
```
Comment (squioc) [inline HarfangLab/harfanglab/ingest/parser.yml:86]: According to the ECS specification, the values for `network.direction` are predefined (cf [ECS Specification](https://www.elastic.co/guide/en/ecs/current/ecs-network.html#field-network-direction)).
I suggest the following change:
```suggestion
      - translate:
        dictionary:
          'in': 'inbound'
          'out': 'outbond'
        mapping:
          json_event.message.direction: network.direction
```

**Review by squioc — COMMENTED** [review]
Thanks for your contribution.

**Comment by ADE-dstny** [comment]
I'm sorry but I don't understand why there is a smart description test, because I didn't change it at all and there already is one.

Let me now if I missed something and I will fix it as soon as possible !

**Comment by squioc** [comment]
@ADE-dstny The smart-description test ensures it exists, at least, one smart-description for each event in the tests.

**Pull request merged**